src/components/Class/ClassCard/index.js

```diff
@@ -176,72 +181,101 @@ function ClassCard({ item, editClass, enroll, style }) {
+        </div>
+        <div>
+          <Typography className={classes.spacing}>
+            Facilitator : {item.facilitator.name}
+          </Typography>
+          <Typography className={classes.spacing}>
+            Date:{moment(classStartTime).format("DD-MM-YYYY")}{" "}
+          </Typography>
+          <Typography className={classes.spacing}>
+            Time:{moment(classStartTime).format("hh:mm a")} -{" "}
+            {moment(classEndTime).format("hh:mm a")}
+          </Typography>
+        </div>
+        <Grid
+          container
+          spacing={2}
+          sx={{ mt: "50px" }}
+          // style={{ display: "flex", flexDirection: "column" }}
+        >
+          <div className={classes.Buttons}>
+            {!item.enrolled ? (
+              loading ? (
+                <div className="loader-button">
+                  <Loader />
+                </div>
+              ) : (
+                <Button
+                  type="submit"
+                  variant="contained"
+                  onClick={() => {
+                    handleClickOpenEnroll(item.id);
+                    console.log("poo");
+                  }}
+                >
+                  {/* {enroll} */}
+                  Enroll
+                </Button>
+              )
+            ) : loading ? (
               <div className="loader-button">
                 <Loader />
               </div>
             ) : (
-              <button
+              <Button
                 type="submit"
-                className={style}
+                variant="contained"
+                color="error"
+                className="class-drop-out"
                 onClick={() => {
-                  handleClickOpenEnroll(item.id);
+                  handleClickOpenUnenroll(item.id);
                 }}
               >
-                {enroll}
-              </button>
-            )
-          ) : loading ? (
-            <div className="loader-button">
-              <Loader />
-            </div>
-          ) : (
-            <button
-              type="submit"
-              className="class-drop-out"
-              onClick={() => {
-                handleClickOpenUnenroll(item.id);
-              }}
-            >
-              Drop out
-            </button>
-          )}
-          {item.facilitator.email == user.data.user.email || flag ? (
-            <div className="class-card-actions">
-              <i
-                className="class-card-action-icon fa fa-trash"
-                onClick={() => handleClickOpen(item.id)}
-              />
-              <i
-                className="class-card-action-icon class-card-edit fa fa-edit"
-                onClick={() => {
-                  handleEdit(item.id);
-                }}
-              />
-            </div>
-          ) : null}
-        </div>
+                Drop out
+              </Button>
+            )}
+            {item.facilitator.email == user.data.user.email || flag ? (
```

Comment:
Use triple equals when the lefthand and righthand sides are expected to have the same type (`string` in this case) 
```suggestion
            {item.facilitator.email === user.data.user.email || flag ? (
```